data/journal_check/bug_refs.json

```diff
@@ -832,5 +832,40 @@
             "microos":  ["Tumbleweed"]
         },
         "type": "bug"
+    },
+    "bsc#1240428": {
```

Comment:
I would like to remove this from the list. There is a fix already accepted in the codestream, so it should not happen any more.